### PR TITLE
Implement v1 of the healthcheck and create github files for open source

### PR DESCRIPTION
**Pull request by kgapos**

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Node Healthcheck
+
+![Node Hosting Project Logo](images/node-hosting.png)
+
+## Introduction
+
+This is a simple API that checks the health of a Vechain Node by comparing the timestamp of its latest block with the current time. If the timestamp is within an acceptable tolerance setting, it is considered healthy.
```
Comment (fabiorigam) [inline .github/README.md]: Shouldn't it be "vechain", with lowercase v?

**Review by fabiorigam — CHANGES_REQUESTED** [review]
The code in the src folder has no comments. We need to add comments to make the code clearer. I can add them, let me know :)

**Comment by kgapos** [comment]
> The code in the src folder has no comments. We need to add comments to make the code clearer. I can add them, let me know :)

Clean, readable code is usually enough to convey the intention. Failing that, comments are welcome. We also don't have any unit tests, but there's not much business logic here to be tested. Still, I had to relax the SonarCloud quality gate in order to accept a 0% unit test code coverage. Contributions in comments and tests are welcome anytime @fabiorigam, thank you in advance!

**Review by fabiorigam — DISMISSED** [review]

File: .github/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ We expect all community participants (contributors, paid or otherwise; sponsors;
 
 ## 8. Contact Information
 
-You can contact the project maintainer at [email@example.com].
+You can contact the project maintainer at kostas.apostolopoulos@vechain.org.
```
Comment (Rishikeshpal) [inline .github/CODE_OF_CONDUCT.md]: May be add me & fabio too?

**Review by Rishikeshpal — APPROVED** [review]

**Pull request merged**